src/api.rs

```diff
@@ -81,6 +82,9 @@ pub struct EncoderConfig {
   pub train_rdo: bool,
 }
 
+/// Default preset for EncoderConfig, In this DEFAULT_SPEED is set to 5 
+/// and called the with_speed_preset with this value(5). We are using 5 
+/// as it is in the middle so it will have good balance of quality and speed.
```

Comment:
Omit internal information that might or might not change, just mention that the default is a balance between speed and quality and mention that the user could use `with_speed_preset` to have a default structure with a specific speed level.